pkg/crc/preflight/preflight_checks_windows.go

```diff
@@ -53,16 +53,11 @@ func checkWindowsEdition() error {
 	windowsEdition := strings.TrimSpace(stdOut)
 	logging.Debugf("Running on Windows %s edition", windowsEdition)
 
-	if strings.HasPrefix(windowsEdition, "Professional") {
-		return nil
+	if windowsEdition == "Home" {
```

Comment:
This table you link to has a row saying:
> Has N or KN variants? Yes

so I think you need to check for a `Home` prefix, and not just check for `==`.

Does the documentation also needs to be updated?

Reply:
> so I think you need to check for a Home prefix, and not just check for ==.

 will update it, i missed that row, Thanks!

> Does the documentation also need to be updated?

We could add to it, we only mention Home edition is not supported.
> On Microsoft Windows, CodeReady Containers requires the Windows 10 Pro Fall Creators Update (version 1709) or newer. CodeReady Containers does not work on earlier versions or editions of Microsoft Windows. Microsoft Windows 10 Home Edition is not supported.